org.skytemple.SkyTemple.yml

```diff
@@ -0,0 +1,144 @@
+app-id: org.skytemple.SkyTemple
+runtime: org.gnome.Platform
+runtime-version: '42'
+sdk: org.gnome.Sdk
+command: /app/bin/run.sh
+finish-args:
+  - "--share=network"
+  - "--socket=fallback-x11"
+  - "--socket=wayland"
+  - "--filesystem=host"
```

Comment:
it doesn't need `host` to do what you want.

Reply:
Just access to the home directory isn't enough either since I know users will also want to modify files outside of that.
Once https://github.com/flatpak/xdg-desktop-portal/issues/463 gets some traction I would narrow the permissions down more. Until then I don't really see a viable other option, but if you know one I'm happy to change this.